test.sh

```diff
@@ -18,34 +18,49 @@
 set -euo pipefail
 
 DEFAULT_INPUT="src/test/resources/samples/*.txt"
-FORK=${1:-""}
-INPUT=${2:-$DEFAULT_INPUT}
 
-if [ "$#" -eq 0 ] || [ "$#" -gt 2 ] || [ "$FORK" = "-h" ]; then
+if [ "$#" -eq 0 ] || [ "$#" -gt 3 ] || [ "$1" == "-h" ]; then
   echo "Usage: ./test.sh <fork name> [input file pattern]"
   echo
   echo "For each test sample matching <input file pattern> (default '$DEFAULT_INPUT')"
   echo "runs <fork name> implementation and diffs the result with the expected output."
   echo "Note that optional <input file pattern> should be quoted if contains wild cards."
+  echo "Use optional flag --quiet to suppress output."
   echo
   echo "Examples:"
   echo "./test.sh baseline"
   echo "./test.sh baseline src/test/resources/samples/measurements-1.txt"
   echo "./test.sh baseline 'src/test/resources/samples/measurements-*.txt'"
+  echo "./test.sh --quiet baseline"
```

Comment:
This is not needed as caller  can always redirect output as they want. The scripts should stay as simple as possible. This way they are better composable, see https://en.wikipedia.org/wiki/Unix_philosophy

Reply:
I think this
```
./test.sh --quiet baseline 'src/test/resources/samples/measurements-*.txt'
```
won't work and there is no point to add full-fledged flag parsing here.